packages/components/addon/components/hds/tabs/index.hbs

```diff
@@ -0,0 +1,34 @@
+{{! template-lint-disable no-invalid-role }}
+{{! template-lint-disable require-context-role }}
+
+<div class={{this.classNames}} ...attributes>
+  <div class="hds-tabs__tablist-wrapper">
+    <ul class="hds-tabs__tablist" role="tablist">
+      {{yield
+        (hash
+          Tab=(component
+            "hds/tabs/tab"
+            didInsertNode=this.didInsertTab
```

Comment:
@alex-ju I see in your recent code you've used `onInsert` - see here: https://github.com/hashicorp/design-system/blob/79047c1c088e1cbc40e85a89a92a108c75346ec1/packages/components/addon/components/hds/form/field/index.hbs#L18 and how it's consumed here https://github.com/hashicorp/design-system/blob/79047c1c088e1cbc40e85a89a92a108c75346ec1/packages/components/addon/components/hds/form/field/index.js#L16-L18
while @KristinLBradley has used `didInsertNode`. I imagine/remember you did some testing/explorations at the time: what made you land on `onInsert` (advantages)? do you think we should use it also here (for consistency, maybe)? or maybe they're the same thing (same function, two different names?)

Reply:
OK, never mind: I just realized that it's just a different name function, but both use `{{did-insert ...}}` as invocation. 🤦‍♂️